pkg/helidonapp/helidonapp.go

```diff
@@ -144,8 +144,8 @@ func CreateHelidonAppCR(mcName string, namespace string, app *v1beta1v8o.Verrazz
 	return &helidonApp
 }
 
-// CreateDeployment creates a deployment for the verrazzano-helidon-app-operator.
-func CreateDeployment(namespace string, labels map[string]string, image string) *appsv1.Deployment {
+// CreateAppOperatorDeployment creates a deployment for the verrazzano-helidon-app-operator.
+func CreateAppOperatorDeployment(namespace string, labels map[string]string, image, memoryRequest string) *appsv1.Deployment {
 	deployment := &appsv1.Deployment{
```

Comment:
So does the string type apply to both image and memoryRequest args?  I expected to see 
```image string, memoryRequest string```

Reply:
Yes, go allows multiple arguments of same type to be declared this way - x, y, z string